src/styles/blocks/wrapper.scss

```diff
@@ -0,0 +1,5 @@
+.wrapper {
+  @include flex-center;
+  width: 100vw;
```

Comment:
100vh is fine however don't use 100vw, if your site had a vertical scroll then this would cause a horizontal overflow / scroll. You can check how it behaves by setting on any sites wrapper `width: 100vw;` in the devtools in your browser (even here on github)